src/test/common/steps/enter-code-steps.js

```diff
@@ -23,3 +23,9 @@ Then(/^I am informed that I must enter in the code that was sent to me$/, async
     pages.enterCode.getConfirmationCodeLinkErrorCss(),
     'Enter the 6 digit code we sent you')
 })
+
+Then(/^The request a new code link points to the send code page$/, async function () {
+  const requestNewCodeLink = await pages.enterCode.getRequestNewCodeLink()
+  const requestNewCodeLinkHref = await requestNewCodeLink.getAttribute('href')
+  expect(requestNewCodeLinkHref).to.be.equal(`${pages.url}/send-code`)
```

Comment:
You should be getting the `/send-code` link from the SendCode page object.

Reply:
Updated to use `pages.sendCode.getPath()`